src/azure-cli/azure/cli/command_modules/acs/tests/latest/test_aks_commands.py

```diff
@@ -7923,6 +7924,69 @@ def test_aks_nodepool_snapshot(self, resource_group, resource_group_location):
             self.is_empty()
         ])
 
+    @AllowLargeResponse()
+    @AKSCustomResourceGroupPreparer(
+        random_name_length=17, name_prefix="clitest", location="westus2"
+    )
+    def test_aks_nodepool_delete_machines(self, resource_group, resource_group_location):
```

Comment:
Queued [live test](https://dev.azure.com/msazure/CloudNativeCompute/_build/results?buildId=103484031&view=results) for this, you'll need to commit the recording file (would be generated by running the test case in live mode, find it from pipeline artifact) to pass CI.

Reply:
https://dev.azure.com/msazure/CloudNativeCompute/_build/results?buildId=103633419&view=results

This is the passed test I ran. Wondering which recording file do I need? btw, I uploaded the [test_aks_nodepool_delete_machines.yaml](https://github.com/Azure/azure-cli/pull/29921/files#diff-fea7674872e9f4480de5f8cafaa4bdb97199cea7b12e6f20e35feef77cbbcea9) already.

<img width="377" alt="image" src="https://github.com/user-attachments/assets/deba74cb-606d-48b2-ab1e-c3aacae2041b">

Reply:
Take another look, it seems that you performed some data plane operations in the test case, which cannot be properly replayed. Please mark the case as `@live_only()` (and left a comment, remove the corresponding recording file) to bypass the check.

Reply:
done